kit/inputs/src/UIKeyTextView.tsx

```diff
@@ -13,12 +13,18 @@ import {
 const MAX_KEY_LENGTH = 64;
 
 type OnDone = (key: string) => void | Promise<void>;
+type onChange = (key: string) => void | Promise<void>;
+type OnSuccess = (success: boolean) => void | Promise<void>;
+type OnError = (error: boolean) => void | Promise<void>;
 
 export function useKeyTextView(
     ref: React.Ref<TextInput> | null,
     isFocused: boolean,
     props: UIMaterialTextViewProps & {
         onDone: OnDone;
+        onSuccess?: OnSuccess;
+        onError?: OnError;
+        onChangeKey?: onChange;
```

Comment:
You could've just pass `onChangeText`, it should be done automatically, without any additional code
https://github.com/tonlabs/UIKit/blob/7edd865e1179ff18d6cdc64ede5f465d4f1af23d/kit/inputs/src/UITextView.tsx#L132
https://github.com/tonlabs/UIKit/blob/7edd865e1179ff18d6cdc64ede5f465d4f1af23d/kit/inputs/src/UITextView.tsx#L184

Reply:
Yep, it works, but this prop excludes from the original component.

Reply:
Yeah, probably need to sth with types to allow to pass it there :)

Reply:
https://github.com/tonlabs/UIKit/pull/1476/commits/9a0447c5cc35417b0ecb3622a1aaebf571c8e06b